File: src/targets/csharp/restsharp/fixtures/cookies.cs
<<<<<<< HEAD
var client = new RestClient("https://httpbin.org/cookies");
var request = new RestRequest(Method.GET);
=======
var client = new RestClient("http://mockbin.com/har");
var request = new RestRequest(Method.POST);
>>>>>>> upstream/master
request.AddCookie("foo", "bar");
request.AddCookie("bar", "baz");
IRestResponse response = client.Execute(request);

File: src/targets/javascript/axios/fixtures/cookies.js
import axios from 'axios';

const options = {
<<<<<<< HEAD
  method: 'GET',
  url: 'https://httpbin.org/cookies',
  headers: {cookie: 'foo=bar; bar=baz'}
};

axios
  .request(options)
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.error(error);
  });
=======
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: {cookie: 'foo=bar; bar=baz'}
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master
